src/app/api/events/mockData/route.ts

```diff
@@ -0,0 +1,58 @@
+// @ts-nocheck
+
+import type { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method === "POST") {
```

Comment:
@SachiGoto Thank you for providing a new idea! 
You are using API routes for the page router, which is one causal issue of failing to fetch data.
You can find the syntax for the App router [here](https://nextjs.org/docs/app/building-your-application/routing/route-handlers).

Reply:
Ah @Lada496, you are so right! I followed the documentation you attached above and write an api call differently and it worked! 

<img width="1326" alt="Screenshot 2023-11-23 at 2 38 17 PM" src="https://github.com/van-squad/VanTechEventFinder/assets/70562492/53dd75e8-0e3a-4c57-a936-8e3c40265eea">
I wouldn't have been able to figured it out without you!

Reply:
@SachiGoto amazing!